src/ctap/data_formats.rs

```diff
@@ -1177,23 +1128,23 @@ mod test {
     #[test]
     fn test_from_into_cred_protection_policy() {
         let cbor_policy = cbor::Value::from(CredentialProtectionPolicy::UserVerificationOptional);
-        let policy = CredentialProtectionPolicy::try_from(&cbor_policy);
+        let policy = CredentialProtectionPolicy::try_from(cbor_policy.clone());
         let expected_policy = CredentialProtectionPolicy::UserVerificationOptional;
         assert_eq!(policy, Ok(expected_policy));
         let created_cbor = cbor::Value::from(policy.unwrap());
         assert_eq!(created_cbor, cbor_policy);
 
-        let cbor_policy_error = cbor_int!(-1);
-        let policy_error = CredentialProtectionPolicy::try_from(&cbor_policy_error);
+        let cbor_policy_error: cbor::Value = cbor_int!(-1);
```

Comment:
Out of curiosity, why do we need to specify the type?

Reply:
The compiler complained, so I was assuming the macro does not uniquely specifiy the type.

Reply:
Oh right, having to specify the type is actually a feature: https://github.com/google/OpenSK/blob/642ee69a9bd6939d4f05c298fcfea101a7dc265a/libraries/cbor/src/macros.rs#L129-L130. Good to know.